.eslintrc.js

```diff
@@ -20,6 +20,21 @@ module.exports = {
   root: true,
   rules: {
     'no-undef': 'off',
+    'no-restricted-syntax': [
+      'error',
+      {
+        selector: 'ForInStatement',
+        message: 'for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.',
+      },
+      {
+        selector: 'LabeledStatement',
+        message: 'Labels are a form of GOTO; using them makes code confusing and hard to maintain and understand.',
+      },
+      {
+        selector: 'WithStatement',
+        message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
+      },
+    ],
```

Comment:
We should consider splitting this out in to another PR. Perhaps taking on the challenge of maintaining our own standards rather than inheriting someone else's. (see: https://github.com/libero/article-store/pull/114#issuecomment-565065488)

Reply:
I'm not sure if we can simply disable that one rule, I think we are going to need to explicitly list all of the rules we want to follow in `no-restricted-syntax`.

Reply:
To delay the linting change, let’s try something like https://www.npmjs.com/package/async-iterator-all